lib/migration/manager/manager.go

```diff
@@ -75,9 +75,9 @@ func (m *DataMigrator) Register(ver int, target string, plugin Plugin) error {
 			Message: "Empty Plugin object is not allowed",
 		}
 	}
-	if ver > version.MaxPluginVersion {
+	if ver > feature.MaxPluginVersion {
```

Comment:
I left this like that for now as this is the only caller, we can add a fn if we find ourselves doing this check over and over again :)